proto/elys/tokenomics/time_based_inflation.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package elysnetwork.elys.tokenomics;
+
+import "elys/tokenomics/inflation_entry.proto"; 
+
+option go_package = "github.com/elys-network/elys/x/tokenomics/types";
+
+message TimeBasedInflation {
+  uint64 startBlockHeight = 1; 
+  uint64 endBlockHeight = 2; 
```

Comment:
@cosmic-vagabond If we're going to be handing inflation through the epochs module won't we need a time duration rather than an endblock?

Reply:
@AustinoBombino for this module I was thinking of not relying on the epoch module but rather just use the startBlockHeight to calculate which Yearly inflation values should be used.